model/roles.go

```diff
@@ -64,11 +64,12 @@ type RoleManifest struct {
 
 // JobReference represents a job in the context of a role
 type JobReference struct {
-	*Job              `yaml:"-"`                 // The resolved job
-	Name              string                     `yaml:"name"`    // The name of the job
-	ReleaseName       string                     `yaml:"release"` // The release the job comes from
-	ExportedProviders map[string]jobProvidesInfo `yaml:"provides"`
-	ResolvedConsumers map[string]jobConsumesInfo `yaml:"consumes"`
+	*Job                `yaml:"-"`                 // The resolved job
+	Name                string                     `yaml:"name"`    // The name of the job
+	ReleaseName         string                     `yaml:"release"` // The release the job comes from
+	ExportedProviders   map[string]jobProvidesInfo `yaml:"provides"`
+	ResolvedConsumers   map[string]jobConsumesInfo `yaml:"consumes"`
+	ContainerProperties JobContainerProperties     `yaml:"properties"`
```

Comment:
It needs some kind of prefix because it uses struct embedding: https://github.com/SUSE/fissile/blob/f453dc862ca86700e16c29b770d914bddf18e58e/model/roles.go#L67 and Job already has a Properties field.